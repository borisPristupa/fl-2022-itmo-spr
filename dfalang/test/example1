Alphabet: A B alpha
States: C D beta
Start: beta
Terminals: beta D
beta -> C: A
beta -> D: alpha
C -> D: B alpha
