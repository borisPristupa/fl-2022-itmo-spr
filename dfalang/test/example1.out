Alphabet: A B alpha   
States: C D beta   
Start: beta    
Terminals: D beta
C -> D: B alpha
beta -> C: A
beta -> D: alpha
